src/Main.java

```diff
@@ -1,5 +1,79 @@
 public class Main {
-    public static void main(String[] args) {
-        System.out.println("Hello world!");
+    public String missingChar(String str, int n) {
+        return str.substring(0, n) + str.substring(n + 1);
+    }
+
+    public int sumDouble(int a, int b) {
+        return a + b;
+    }
+
+    public int intMax(int a, int b, int c) {
+        return intMax(a, b, c);
+    }
+
+
+    public boolean nearHundred(int n) {
+        if (n >= 90 && n <= 110 || n >= 190 && n <= 210) {
```

Comment:
Correct, however, you could have used Math.abs() method as it was said in description.
Then it would be 
```
if (Math.abs(100 - n)  <= 10 || Math.abs(200 - n)  <= 10) {
            return true;
        } else {
            return false;
        }
```